charts/opentelemetry-collector/values.yaml

```diff
@@ -176,6 +176,17 @@ presets:
     #       value: http
     #     firstEntryRegex: ^[^\s].*
     #     combineWith: ""
+
+    # Configures the collector to reduce log attributes by removing specified log record attributes.
+    # This sub-preset adds a transform processor to the logs pipeline.
+    reduceLogAttributes:
+      enabled: false
+      denylist:
+        - "log.iostream"
+        - "log.file.path"
+        - "logtag"
+        - "time"
```

Comment:
Do not this break the UI? Because we are showing the time when the log was emitted. Or is is maybe taken from other place?